ci/README.md

```diff
@@ -1,5 +1,5 @@
 ## Overview
-Travis is enabled in this repository so each pull request is checked before being allowed to merge.
+GitHub Actions is enabled in this repository so each pull request is checked before being allowed to merge.
 The system is based on `fiware/orion-ci:rpm7` which is built from master branch each time a new PR lands in master,
```

Comment:
Fixing this file I have found this paragraph:

> Note that `fiware/orion-ci:rpm7` is *not* rebuilt due to changes in the PR branch under test. Thus, if you are developing a functionality that requires a new library or base system you need to do *first* a PR adding such library or base system to `ci/rpm7/build-dep.sh` and/or `Dockerfile`. Once that PR gets merged into master and `fiware/orion-ci:rpm7` gets rebuild  (checking progress in Dockerhub at: https://hub.docker.com/r/fiware/orion-ci/builds) your PR branch with the new  functionality is ready to be tested with travis. 

I think the change to GitHub Actions doesn't change that and the text is still correct. @wistefan could you confirm, please?

Reply:
Yes, its still as described.

Reply:
NTC